tests/sav/hello_ios.res

```diff
@@ -0,0 +1 @@
+hello_ios.bin is a directory
```

Comment:
Compiled iOS apps are directories, so this res indicates a successful compilation.